recipes-kernel/kernel-tests/kernel-performance-tests.bb

```diff
@@ -12,8 +12,8 @@ S = "${WORKDIR}"
 
 DEPENDS = "virtual/kernel"
 RDEPENDS:${PN}-ptest += "bash rt-tests fio iperf3 python3 python3-pip"
-RDEPENDS:${PN}-ptest:append:x64 += "fw-printenv"
-RDEPENDS:${PN}-ptest:append:armv7a += "u-boot-fw-utils"
+RDEPENDS:${PN}-ptest:append:x64 = " fw-printenv"
+RDEPENDS:${PN}-ptest:append:armv7a = " u-boot-fw-utils"
```

Comment:
Check.

Reply:
From `bitbake -e kernel-performance-tests`, `RDEPENDS` contains `fw-printenv` and doesn't contain `u-boot-fw-utils`.